src/Microsoft.AspNet.Mvc.Core/Filters/ProducesAttribute.cs

```diff
@@ -17,6 +17,21 @@ namespace Microsoft.AspNet.Mvc
     [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
     public class ProducesAttribute : ResultFilterAttribute, IApiResponseMetadataProvider
     {
+        /// <summary>
+        /// Initializes an instance of <see cref="ProducesAttribute"/>.
```

Comment:
@harshgMSFT - I'm not aware of a consensus here, both seem pretty correct. If you think this is important let's dicuss after standup sometime or consult with docs folks and get a 'right' answer.

Very likely all of this will be reviewed at some point and made consistent then.